### ISSUE TITLE
AI settings + model availability

### ISSUE THREAD

**Issue by stevenorechow** [workeffortwaste/horseman#128]
Hey Chris, it would be awesome if you could implement some options in the AI settings to define things like endpoint, model, and API provider so that we have some options and more granularity. 

That way, even if you stop developing Horseman, people who are paying for it can use newer models like gpt-4o or Claude Sonnet 3.5.

Cheers.

**Comment by davidldennison** [comment]
This would be great!

**Comment by stevenorechow** [comment]
@davidldennison development has been completely stagnant for some time now, I wouldn't count on it anytime soon unfortunately. Lots of other alternatives now though!

**Comment by davidldennison** [comment]
@sorec574 Yeah unfortunately. I haven't found any alternatives I like. Mind sharing some that you've found?

**Comment by jblifestyles** [comment]
@workeffortwaste  .. any plans to update this ??
